### PR TITLE
french language updates

### PR DESCRIPTION
**Pull request by tariqbenezza**
French language correction

Thank you !

### CODE REVIEW

**Comment by lexoyo** [comment]
You did not merge it?

File: share/hedgewars/Data/Locale/hedgewars_fr.ts [hunk 1]
```diff
@@ -112,23 +112,23 @@
     </message>
```
Comment (antonc27) [inline share/hedgewars/Data/Locale/hedgewars_fr.ts]: Test comment

**Comment by antonc27** [comment]
@lexoyo It is still open. Was auto-closed by GitHub due to some repo cleaning.

**Comment by antonc27** [comment]
Was accepted by cherry-picking actual commit.

**Pull request closed without merging**